### DISCUSSION TITLE
Can this be used to take/migrate an existing movie (or TV show) Plex collection from one sever to another?

### DISCUSSION THREAD

**Discussion by oshura3** [Taxel/PlexTraktSync#842] [Q&A]
Hello there,

Recently, I had to spin up a new plex server and for reasons, I don't have the same exact media on my new server as I did on my old one. As a result, I wasn't able to do a direct database copy for my new server. I was hoping to migrate some of the collections I had built in my old server to my new one using this "app"/plugin/script (?) and I have a pretty strong suspicion that I should be able to do that, I just don't know how to configure everything. Would I have to manually add all of the shows and/or movies to a list in trakt so they are then added to the new server via the liked watchlist option, or would I be able to somehow use the script to create a new watchlist based on the collections on my old server and then use those newly created watchlists to make new collections on the new server? Thanks a bunch in advance. If this question needs further clarification, I would be glad to provide it.

Cheers

TL;DR: I'm going to attempt to restate the issue here for clarity:
1. I have existing plex collections of movies and TV shows in an old instance that I won't be able to access soon.
2. I don't have the exact same movies or shows on the new server I'll be using as on the old one that's sunsetting.
3. I would like to somehow use this script to copy/migrate the collections from the old server to the new server so that I don't have to manually create all of the collections (which would take quite a long time).
4. Help would be truly appreciated!

**Comment by simonc56** [comment]
This script doesn't handle Plex [Collections](https://support.plex.tv/articles/201273953-collections/) (yet).
- https://github.com/Taxel/PlexTraktSync/discussions/134

**Comment by glensc** [comment]
I'm not even sure the user here understands Plex Collection as a term, or just names their Plex Movile Library as Plex Collection.
